nameko/timer.py

```diff
@@ -76,9 +81,14 @@ def handle_timer_tick(self):
         # Note that we don't catch ContainerBeingKilled here. If that's raised,
         # there is nothing for us to do anyway. The exception bubbles, and is
         # caught by :meth:`Container._handle_thread_exited`, though the
-        # triggered `kill` is a no-op, since the container is alredy
+        # triggered `kill` is a no-op, since the container is already
         # `_being_killed`.
-        self.container.spawn_worker(self, args, kwargs)
+        self.container.spawn_worker(
+            self, args, kwargs, handle_result=self.handle_result)
+
+    def handle_result(self, worker_ctx, result, exc_info):
+        self.event.send(result, exc_info)
```

Comment:
You don't actually need to pass `result` and `exc_info` in here. It would be sufficient to send `True`.